site/content/docs/developing/domain.en.md

```diff
@@ -2,9 +2,9 @@
 
 !!!attention
     Today, a Route 53 domain name can be associated only when running `copilot app init`.  
-    If you'd like to update your application with a domain ([#3045](https://github.com/aws/copilot-cli/issues/3045)), 
-    you'll need to initialize a duplicate app with `--domain` and then run `copilot app delete` to 
-    remove the old one.
+    If you'd like to update your application with a domain ([#3045](https://github.com/aws/copilot-cli/issues/3045)),
+    you'll need to initialize a duplicate app with `--domain` and then run `copilot app delete` to
+    remove the old one. However, you can import validated ACM certificates that include your existing domains to your environment.
```

Comment:
Now that we are back at it😂 can they really initialize a duplicate app without deleting it first?

Reply:
On a different topic, what do you think of reorganizing it as something like:
```
There are two ways to use custom domains for your application:
1. `copilot app init --domain`, this will blahblahblah and Copilot will create and manage a certificate for you blahblah
2. Bring your certificate by the import feature blahblah
```